src/main/java/dk/frankbille/scoreboard/domain/User.java

```diff
@@ -26,6 +26,7 @@
 	private String username;
 	private Player player;
 	private League defaultLeague;
+	private RatingCalculatorType ratingCalculator;
```

Comment:
Good idea. But ratings are also used when displaying non-league related game statistics: when displaying played games on a player-page. Which rating-calculator should be sued for non-league related pages?
It could also just be a system-wide settings? I just though having it as an opt-in user-option would avoid too much user-confusion.